tests/integration/test_web_client.py

```diff
@@ -51,6 +51,12 @@ def test_creation_lifecycle(self, antares_web: AntaresWebDesktop):
 
         study = create_study_api("antares-craft-test", "880", api_config)
 
+        actual_study = read_study_api(api_config, study.service.study_id)
+
+        assert study.service.study_id == actual_study.service.study_id
+        assert study.name == actual_study.name
+        assert study.version == actual_study.version
```

Comment:
You should also test that the 2 studies contain the same areas name. And the same settings (just test some values if the == doesn't work)